controller/list-bucket.sh

```
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export SECRET_NAME="minio"
+
+# Set your MinIO credentials and configuration
+export AWS_ACCESS_KEY_ID=$(kubectl get secret "$SECRET_NAME" -n "$NAMESPACE" -o jsonpath='{.data.root-user}' | base64 --decode)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)